### PR TITLE
feat(docker): make tags configurable

### PR DESCRIPTION
**Pull request by CherryKitten**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -75,6 +84,7 @@ jobs:
         uses: famedly/metadata-action@v4
         with:
           images: ${{ inputs.registry }}/${{ inputs.image_name }}
+          tags: ${{ inputs.tags }}
```
Comment (jcgruenhage) [inline .github/workflows/docker.yml:87]: I'm not sure how this magic works exactly, but I *think* we also need this in the docker metadata step in the merge job

Comment (CherryKitten) [reply .github/workflows/docker.yml:87]: yep, I'm pretty sure you're right, I just forgot about the repeated metadata step..

**Review by jcgruenhage — APPROVED** [review]
lgtm. Before this is merged, I'd like to see this tried out, but considering we're currently just moving the default of the action into our workflow, I don't really see a way this could break.

**Comment by jcgruenhage** [comment]
has been tested in https://github.com/famedly/container-image-ansible/pull/2 successfully, the images are built and the tags are used correctly.

There is one bit that is still problematic in the workflow here: The digests aren't separated by which tag is being built, so if we have multiple tags being built in the same workflow run, they all get merged into each tag. We also had this with powerdns, when we built for multiple registries. I think the solution for this is adding `${{ inputs.registry }}/${{ inputs.image_name }}/<tag after it's been resolved>` or something to the artifact path, which would ensure that these artifacts don't clash. The last bit is a bit more complicated, as it needs to get the tag out of the docker metadata step somehow.

That can be added in a follow-up PR though, as that's a bug that isn't directly related to configurable tags and that was present before as well.

**Pull request merged**